assets/000-postgres.prep

```diff
@@ -117,3 +117,7 @@ foreground {
   s6-setuidgid postgres
   /usr/local/bin/pulpcore-manager migrate
 }
+pipeline {
+  echo "ready"
+}
+tee /database/status
```

Comment:
This does not seem to be used. (I would have preferred that method.)

Reply:
it is used here: https://github.com/pulp/pulp-oci-images/blob/latest/assets/wait_on_database_migrations.sh#L11

Reply:
Oh right, but that line has no consequences for any of the services.